example/lib/src/counters/bloc_counter/bloc_counter_bloc.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:surf_widget_test_composer_example/src/counters/bloc_counter/bloc_counter_event.dart';
+
+class BlocCounterBloc extends Bloc<BlocCounterEvent, int> {
```

Comment:
It seems that one `bloc` of mentions in the title is enough

Reply:
This is because of the the structure
<img width="339" alt="image" src="https://github.com/surfstudio/flutter-surf-widget-test-composer/assets/23572568/c3cf603b-271d-490f-8fe7-34c88b5b3c4e">